src/interface/src/app/login/login.component.ts

```diff
@@ -48,6 +51,14 @@ export class LoginComponent {
     });
   }
 
+  resetPassword() {
+
+    // TODO: sterlingwellscaffeine -- add the functionality that validates the 
+    // email. The reason that this is not included in this CL is that I would like
+    // to make that a part of the server call and the server is not yet up and running.
+    this.dialog.open(ResetPasswordDialog);
```

Comment:
Do we want to leave users on this page or redirect them to the homepage or somesuch (as they'll come back to the app via their email link, so it doesn't make sense to me to leave this login page open)

Reply:
Did the demo help explain the functionality to you? The user is never leaving the homepage.

Reply:
Could you paste demo screenshots in the PR description?